docker-compose.yml

```diff
@@ -2,14 +2,15 @@ version: '3'
 
 services:
   postgrest:
-    image: "postgrest/postgrest:v10.1.2"
+    image: "postgrest/postgrest:v12.2.3"
```

Comment:
Great! Gotta love recent versions.

Although maybe we should do that in a separate PR in case we have some debug to do?

Reply:
If we want it in a separate PR, it will have to be merged upstream because if I understood it well aggregation funcs were not available before v12

Reply:
Understood, then I guess that's ok!